RadixWallet/Features/ImportMnemonic/ImportMnemonic+View.swift

```diff
@@ -148,8 +148,23 @@ extension ImportMnemonic {
 							.padding(.horizontal, .medium2)
 							.padding(.bottom, .medium3)
 						} else {
-							Button("DEBUG Acc.Rec.Scan.mne (24)") {
-								viewStore.send(.debugUseTestingMnemonicWithActiveAccounts)
+							if !(viewStore.isWordCountFixed && viewStore.wordCount == .twentyFour) {
```

Comment:
Why do we only want to show it when wordcount is 24?

Reply:
No this code says "don't allow using a 15 word mnemonic as Babylon Device Factor Source" - since BDFS requires 24words